compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -39,7 +39,11 @@
 #include "llvm/TargetParser/Host.h"
 #endif
 #include "llvm/Support/TimeProfiler.h"
+#if LLVM_VERSION_GE(20, 0)
+#include "llvm/Transforms/Utils/Instrumentation.h"
+#else
 #include "llvm/Transforms/Instrumentation.h"
+#endif
```

Comment:
Do we need the include here at all? This file already includes `LLVMWrapper.h`.

Or vice versa, if the include is not really needed in the common header, it could just live here.

Reply:
Good catch, cleaned that up.